doc/nrf/releases_and_maturity/migration/migration_guide_2.7.rst

```diff
@@ -59,7 +59,16 @@ Samples and applications
 
 This section describes the changes related to samples and applications.
 
-|no_changes_yet_note|
+Matter
+------
+
+.. toggle::
+
+   * For the Matter samples and applications:
```

Comment:
Wonder if this should be in the migration guide (which I believe should contain changes that affect customers' applications). This PR changes our samples and applications but customers can still use the previous configuration scheme or whatever they currently use, right?

Reply:
I feel it is okay in the recommended changes section, but am not really certain on that technical side of things.

Reply:
Ah, ok. If this section is for optional changes that may be applied to align with our samples and applications then it's fine with me. I kind of assumed this document should refer to customers' applications only and contain normative language (what you should or must do) rather than describe what we have changed in our samples but I might have misunderstood the purpose.

Reply:
There's a Required changes section that comes first, which are changes the customers must make when updating to the corresponding version of the NCS, and a Recommended changes section that this entry is in.

Reply:
That's right, if they use their configurations it doesn't change anything.

Reply:
Apart from that, we changed our sample's layout so they may have conflicts after updating to NCS 2.6.0 I would leave those entries in the migration notes.